spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -372,16 +375,28 @@ will emit a message with a TaskLaunchRequest object containing the required laun
 would be to launch the timestamp task once every 5 seconds, the stream to implement this would look like:
 
 ```
-stream create foo --definition "triggertask --triggertask.uri=maven://org.springframework.cloud.task.app:timestamp-task:jar:1.0.0.BUILD-SNAPSHOT --trigger.fixed-delay=5 | task-launcher-local" --deploy
+stream create foo --definition "triggertask --triggertask.uri=maven://org.springframework.cloud.task.app:timestamp-task:jar:1.2.0.RELEASE --trigger.fixed-delay=5 | task-launcher-local" --deploy
 ```
 
-=== Translator
+The `triggertask` can be added to the available sources by executing the app register command as follows (for the Rabbit Binder):
```

Comment:
I'll move this to before the stream definition, otherwise if you are just reading along and cut-n-pasting you will get an error.